### PR TITLE
Minor Changes and added funds

### PR DESCRIPTION
**Pull request by Deadify**

### CODE REVIEW

**Comment by Deadify** [comment]
> – Please add the updated nav again
> – Update formatting rules (1 link per country = no need to specify org, 2 or more links per country = please specify org for all)

I addressed all of these, I'm SUPER sorry for the messy sequence of commits! I'll do my best not to let it happen again.

File: Home.jsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+				<Box name="Africa" list_of_links={AFRICA} className="africa" />
+				<Box name="Oceania" list_of_links={OCEANIA} className="oceania" />
+				<Box name="Middle East"	list_of_links={MIDDLE_EAST}	className="middleeast"/>
+			</div>
+		</div>
+	);
+}
+
+export default Home;
```
Comment (mcao) [inline Home.jsx:176]: Why is this file created here/in the root directory?

File: src/components/home_page/Home.jsx [hunk 2]
```diff
@@ -137,41 +154,24 @@ const NorthAmerica = (props) => {
...
+
+				<Box name="Africa" list_of_links={AFRICA} className="africa" />
+				<Box name="Oceania" list_of_links={OCEANIA} className="oceania" />
+				<Box name="Middle East"	list_of_links={MIDDLE_EAST}	className="middleeast"/>
+			</div>
+		</div>
+	);
 }
 
 export default Home;
```
Comment (mcao) [inline src/components/home_page/Home.jsx:177]: Please use the `prettier` extension on Visual Studio Code and lint the code so it complies with formatting standards - and so you're not changing all the code from 2 spaces to tabs every single time you edit a file!

**Comment by mcao** [comment]
uh so I almost screwed everything up but the commits are now in master and everything seems good?? 👍

**Pull request closed without merging**